keps/7610-default-local-queue-creation/README.md

```diff
@@ -0,0 +1,389 @@
+   from `spec.defaultLocalQueue.name` already exists.
+5. If the `LocalQueue` does not exist, the controller will create it. The new
+   `LocalQueue` will reference the current `ClusterQueue` (via
+   `spec.clusterQueue` field) and will include the identifying labels and
+   annotations.
+6. If a `LocalQueue` already exists, do nothing and emit a warning event.
+
+### Admission Webhook
+
+Add new validation admission logic to the existing `ClusterQueue` webhook to
+prevent selector overlap.
+
+1. The webhook triggers on `CREATE` and `UPDATE` of `ClusterQueue` resources.
+2. If `spec.defaultLocalQueue` is not set, the validation is skipped.
+3. If set, the webhook lists all other `ClusterQueue`s in the cluster.
+4. It compares the `namespaceSelector` of the incoming `ClusterQueue` with every
+   other `ClusterQueue` that also has `defaultLocalQueue` enabled.
+5. If a selector overlap is detected and the `defaultLocalQueue.name` is the same,
+   the request is rejected with an error detailing the conflict. This prevents
+   two `ClusterQueues` from attempting to manage the same `LocalQueue` resource.
+
+### Test Plan
+
+<!--
+**Note:** *Not required until targeted at a release.*
+The goal is to ensure that we don't accept enhancements with inadequate testing.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations). Please adhere to the [Kubernetes testing guidelines][testing-guidelines]
+when drafting this test plan.
+
+[testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
+-->
+
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes
+necessary to implement this enhancement.
+
+##### Prerequisite testing updates
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
+
+#### Unit Tests
+
+<!--
+In principle every added code should have complete unit test coverage, so providing
+the exact set of tests will not bring additional value.
+However, if complete unit test coverage is not possible, explain the reason of it
+together with explanation why this is acceptable.
+-->
+
+<!--
+Additionally, try to enumerate the core package you will be touching
+to implement this enhancement and provide the current unit coverage for those
+in the form of:
+- <package>: <date> - <current test coverage>
+
+This can inform certain test coverage improvements that we want to do before
+extending the production code to implement this enhancement.
+-->
+
+
+
+#### Integration tests
+
+<!--
+Describe what tests will be added to ensure proper quality of the enhancement.
+
+After the implementation PR is merged, add the names of the tests here.
+-->
+
+### Graduation Criteria
+
+<!--
+
+Clearly define what it means for the feature to be implemented and
```

Comment:
Here add sections for 

Alpha:
- feature disabled by default
- creation of the LocalQueue which matches the namespaceSelector (no need for conflict prevention)

Beta:
- feature enabled by default
- re-evaluate the strategies for conflict prevention